doc/getting_started/index.rst

```diff
@@ -38,6 +38,11 @@ itself.
 Bootstrap west
 ==============
 
+.. warning::
+
+   It's possible to use Zephyr without installing west, but you have
```

Comment:
Shouldn't this be replacing the block at line 33 in this file, or at least amending it?

Reply:
I missed that, thanks!